models/tmp/stg_zendesk__ticket_schedule_tmp.sql

```diff
@@ -0,0 +1,23 @@
+{%- set source_relation = adapter.get_relation(
+      database=source('zendesk', 'ticket_schedule').database,
+      schema=source('zendesk', 'ticket_schedule').schema,
+      identifier=source('zendesk', 'ticket_schedule').name) -%}
+
+{% set table_exists=source_relation is not none  %}
+
+{% if table_exists %}
+{{ log("Table exists", info=True) }}
+
+select * 
+from {{ source('zendesk', 'ticket_schedule') }}
```

Comment:
Could not use ref or else the conditional would fail (as the table may not exist) therefore, this was needed in order for it to work.

Reply:
Makes sense, however I have a question about how/if this will work if the customer's schema is not `zendesk`

Reply:
That's a good question and one I did not think of. Thankfully, I just tested this on a dataset with a schema different than `zendesk` (it also had a `ticket_schedules` table populated) and the different schema name worked and the `stg_zendesk__ticket_schedule` table was correctly populated with values rather than the one row of nulls!

I am happy to walk you through this test if you would like! It seems setting the `zendesk_schema` variable ensures this will work.